module_utils/certificate/providers/base.py

```diff
@@ -36,10 +37,14 @@ def load_from_params(cls, module, params):
         # pylint: disable=protected-access
         cert_like = cls(module)
 
-        map_attrs = ["dns", "ip"]
+        map_attrs = ["dns", "ip", "email"]
         info = {k: v for k, v in params.items() if k in map_attrs}
 
         info["common_name"] = cert_like._get_common_name_from_params(params)
+        if info.get("ip"):
```

Comment:
this works, but the typical python idiom is to use `if "ip" in info` since you are not otherwise using the result of `get()`

Reply:
I actually want to get the value and verify that's not `None` neither `[]`.

Reply:
ah, ok - then what you have is shorter and more readable then `if "ip" in info and info["ip"]`

Reply:
@richm alright. Although actually a little longer the intention is obviously a lot clearer in that form. I'll change.

Reply:
I think you should leave it as `if info.get("ip"):` - I think that is shorter and more readable than the alternative (imo)

Reply:
I think that if I had `if "ip" in info and info["ip"]` you probably wouldn't have the doubt you had. That's why I assumed the intention would be clearer in the longer form.
But this is so small that I'm ok with either way.